README.md

```diff
@@ -99,7 +99,7 @@ This project relies on a backend service to fetch and handle data. To ensure the
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link]()
+- [Live Demo Link](https://fmanimashaun.github.io/book-car)
```

Comment:
- [ ] When I try to run the project according to instructions, I get this:

![image](https://github.com/fmanimashaun/book-car/assets/57421823/6ee982fc-0017-4a4f-a0da-c83ab26e6504)

![image](https://github.com/fmanimashaun/book-car/assets/57421823/76c2d2a8-0901-4b86-ad6b-07e87aa6d2aa)

![image](https://github.com/fmanimashaun/book-car/assets/57421823/26542e05-0f5f-4d96-850c-aa6dec393177)

Kindly, take a look so your project works as expected.  If there are any special procedures to be done in order to run your project correctly, kindly add them to your readme file. 👍 

_note: This means I couldn't get access to the project's main functionality.  This will be taken a look at the next review._

Reply:
apologies, we did not update our live link, we will do that now